### DISCUSSION TITLE
The WebAssembly files of Jupyterlite were blocked by the firewalls of our company

### DISCUSSION THREAD

**Discussion by MRYingLEE** [jupyterlite/jupyterlite#1127] [Q&A]
Did you meet such a kind of situation?

I noticed that the WebAssembly files from Google can work normally.

Of course, this is not specific to Jupyterlite. Just wondering whether there is a way for Jupyterlite to be more network friendly.

Any comments?

**Comment by qqdaiyu55** [comment]
@MRYingLEE  Please ref to https://jupyterlite.readthedocs.io/en/latest/howto/pyodide/pyodide.html#

**Comment by MRYingLEE** [comment]
I tried your mentioned solution. It didn't work.

But jupyterlite-xeus-python with Jupyterlite 0.2.x can be accessed normally from my company.

So, the issue has been solved by the new version.

Thanks
